sync-branches/action.yaml

```diff
@@ -64,6 +64,20 @@ runs:
       with:
         git-cliff-args: origin/${{ inputs.base-branch }}..HEAD
 
+    - name: Replace PR number to URL
+      id: replace-pr-number-to-url
+      run: |
+        r=$(echo "$CHANGELOG" | sed -r "s|\(#([0-9]+)\)|("${REPOS%.git}"/pull/\1)|g")
+        # Workaround for https://github.community/t/set-output-truncates-multiline-strings/16852
+        r="${r//'%'/'%25'}"
+        r="${r//$'\n'/'%0A'}"
+        r="${r//$'\r'/'%0D'}"
+        echo ::set-output name=changelog::$r
+      env:
+        CHANGELOG: ${{ steps.generate-changelog.outputs.changelog }}
+        REPOS: ${{ inputs.sync-target-repository }}
```

Comment:
Why did you make this an env?

Reply:
To see easily.

Reply:
Could you explain a bit more in detail?

Reply:
If I don't make `$REPOS` env, ${${{ inputs.sync-target-repository }}%.git} is needed. This is not easy to see for me.

Reply:
Ah, I see. But how about naming it as `REPO_URL` or something like that?

Reply:
Or just `REPO` is okay for me.

Reply:
I think "REPO_URL" is better.  Could you suggest it ?

Reply:
```suggestion
        REPO_URL: ${{ inputs.sync-target-repository }}
```

Reply:
I fixed